OloEngine/src/OloEngine/Scripting/C#/ScriptGlue.cpp

```
@@ -1342,6 +1344,145 @@ namespace OloEngine
         NetworkManager::StopServer();
     }
 
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Dialogue ///////////////////////////////////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    static void DialogueComponent_StartDialogue(UUID entityID)
+    {
+        Scene* scene = ScriptEngine::GetSceneContext();
```

Clang (14.0.6)

[warning] variable 'scene' is not initialized

(cppcoreguidelines-init-variables)